### ISSUE TITLE
Electron App unable to open a new window

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core-npm#766]
We recently discovered that when running WebdriverIO inside an Electron app it is not able to create a [new browser window](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/webdriverio/src/index.ts#L357). The code fails with the error message "No such window". Electron [allows opening windows](https://www.electronjs.org/docs/latest/api/window-open) so I'm not sure if it's the integration between WebdriverIO and Electron or maybe a setting in Electron itself that prevented the code from working.

**Comment by udarrr** [comment]
Yes confirm there is no possibility to open new window in electron that's why approach with just builder.analyze() isn't working at all. But i'm wondering about possible solution. Can we skip opening new window ?

**Comment by straker** [comment]
@udarrr For the time being you can use the [`setLegacyMode(true)` function](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/webdriverio#axebuildersetlegacymodelegacymode-boolean--true) to disable opening a new window. However that function is planned to be removed in 5.0 (currently we have no timeframe of when 5.0 will be released).

**Comment by sujithvakathanam** [comment]
can this issue be looked at in priority please? With setLegacyMode workaround , I can get around this issue. Considering that this method will be removed in 5.0, Would greatly appreciate if this can be  looked at?

**Comment by personalizedrefrigerator** [comment]
This also seems to affect `@axe-core/playwright`.
